inginious/frontend/static/css/INGInious.less

```diff
@@ -75,6 +78,7 @@
             color: #333;
             padding-left: 1rem;
             font-size: 1.1em;
+            margin-right: @breadcrumb-a-right-m;
```

Comment:
This does not provide the same result when there are several items in the breadcrumbs.
What you should actually do (this is intuitive) is to replace the `padding-left: 1rem`s of both `<span/>` and `<a/>` by `padding-right: 1rem`